library/std/src/sys/windows/c/bindings.txt

```diff
@@ -1,7 +1,6 @@
 --out windows_sys.rs
 --config flatten std
 --filter
-// tidy-alphabetical-start
```

Comment:
If this is the file that people actually edit, then isn't it still desirable for it to remain sorted?

Whereas, it doesn't seem important to me whether the generated file is sorted or not.

Reply:
Ah, to be clear this PR now sorts bindings.txt automatically not the generated file. I mean, Rust can do this itself well enough (in this specific case) whereas human contributors have tended to get sorting wrong a fair bit. This is possibly also an argument that tidy should at least produce better error messages than "line not in alphabetical order" but fixing that would be a bigger job 😀

Reply:
Oh, I missed that it was writing it back after sorting, not just in memory for the next stage.